front/src/EmailSettings/EmailSettings.js

```diff
@@ -138,8 +140,26 @@ const EmailSettings = () => {
     setSettings({ ...settings, sendPendingTime: newDate });
   };
 
+  // checks whether the email in sähköpostiasetukset käyttäjä/user field is a tasera email
+  const validateEmail = (email) => {
+    const re = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    return re.test(String(email).toLowerCase()) && email.endsWith('@tasera.fi');
+  };
+
   const handleSubmit = (e) => {
     e.preventDefault();
+    if (!validateEmail(settings.user)) { // checks if the email is a tasera email
+      console.log('Invalid email!');
+      setNotification({ open: true, message: 'Invalid email. Needs to be a tasera.fi email!' });
+      return;
+    } 
+    if (!settings.pass) { // checks if the password field is empty
+      console.log('Password is empty!');
+      setNotification({ open: true, message: 'Password cannot be empty!' });
+      return;
+    }
+    console.log('email was correct');
+    setNotification({ open: true, message: 'Email was correct. Settings saved!' });
```

Comment:
We should add these notification strings to localization (front/src/text.json around line 27).